scss/_tag.scss

```diff
@@ -0,0 +1,71 @@
+// Base class
+
+.tag {
+  @extend .badge;
+  @include font-size($tag-font-size);
+  padding: add($badge-padding-y, 3px) $badge-padding-x add($badge-padding-y, 3px); // Boosted mod
```

Comment:
You can remove all `// Boosted mod` comments in this file because it is already specific to Boosted.